deployments/a11y/image/apt.txt

```diff
@@ -0,0 +1,91 @@
+# Some linux packages for basic terminal work, particularly
+# oriented at users new to Unix/cmd line environments.
+
+# Basic unix tools
+man
+man-db
+manpages-posix
+manpages-dev
+manpages-posix-dev
+
+# Download tools
```

Comment:
A lot of the UNIX cli tools and X11 apps probably aren't needed for Lab accessibility testing, e.g. emacs, nano, jed, mc, zsh, nnn, most of the packages that start with "x", firefox, gedit. Maybe curl/wget and vim are useful for just Doing Stuff, but most everything else won't be useful for testing.

Reply:
@ryanlovett Sounds good. Addressed in the next commit.